s3/pkg/service/object.go

```diff
@@ -410,6 +427,7 @@ func GetObject(ctx context.Context, req *pb.GetObjectInput, stream pb.S3_GetObje
 }
 
 func GetEncObject(ctx context.Context, req *pb.GetObjectInput, stream pb.S3_GetObjectStream, s *s3Service) error {
+
```

Comment:
Format code please

Reply:
did that, ran go fmt multiple times, still looks like this...am i missing something ?
![image](https://user-images.githubusercontent.com/48434725/71891788-59711180-316d-11ea-88b4-49b8aca18969.png)

Reply:
How did you run it?

Reply:
like this...